cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/resource/AwsVolumeResourceBuilder.java

```diff
@@ -226,18 +229,39 @@ private String getAvailabilityZoneFromSubnet(AuthenticatedContext auth, CloudRes
                 .build();
     }
 
-    private Function<VolumeSetAttributes.Volume, CreateVolumeRequest> createVolumeRequest(String snapshotId, TagSpecification tagSpecification,
-            VolumeSetAttributes volumeSet) {
-        return volume -> new CreateVolumeRequest()
-                .withAvailabilityZone(volumeSet.getAvailabilityZone())
-                .withSize(volume.getSize())
-                .withSnapshotId(snapshotId)
-                .withTagSpecifications(tagSpecification)
-                .withVolumeType(volume.getType());
+    private boolean isFastEbsEncryptionEnabled(Group group) {
+        return new AwsInstanceView(group.getReferenceInstanceConfiguration().getTemplate()).isFastEbsEncryptionEnabled();
```

Comment:
Minor, we could introduce a new method to replace `group.getReferenceInstanceConfiguration().getTemplate()` with getReferenceTemplate as it is used at many places.

Reply:
Good idea, will do that in a follow-up PR. 👍

Reply:
See #8482.